src/app/view/components/_common/accordion/accordion.ts

```diff
@@ -93,6 +94,10 @@ export class AccordionGroupComponent implements AfterViewInit, OnDestroy {
         return this._isOpen;
     }
 
+    isFaIcon(icon: string): boolean {
```

Comment:
should this be part of a util lib? where other components can use it?

Reply:
Will be deprecated.